scout/adapter/mongo/query.py

```diff
@@ -114,33 +314,28 @@ def build_query(self, case_id, query=None, variant_ids=None, category='snv'):
             if query.get('hgnc_symbols'):
                 hgnc_symbols = query['hgnc_symbols']
                 mongo_query['hgnc_symbols'] = {'$in': hgnc_symbols}
-                logger.debug("Adding hgnc_symbols: %s to query" %
+                LOG.debug("Adding hgnc_symbols: %s to query" %
                              ', '.join(hgnc_symbols))
 
             if query.get('gene_panels'):
                 gene_panels = query['gene_panels']
                 mongo_query['panels'] = {'$in': gene_panels}
 
-        if query.get('chrom'):
-            chromosome = query['chrom']
-            mongo_query['chromosome'] = chromosome
-            #Only check coordinates if there is a chromosome
-            if (query.get('start') and query.get('end')):
-                mongo_query['position'] = {'$lte': int(query['end'])}
+        return gene_query
 
-                mongo_query['end'] = {'$gte': int(query['start'])}
 
-        mongo_query_minor = []
```

Comment:
minor = secondary, major = primary. Got it! :wink:

Reply:
Yes, that's the first thing I should have written in the comment, probably!